app/views/eventDetails/paymentOptions.html

```diff
@@ -228,6 +228,14 @@ <h3 class="panel-title" translate>Cru Event Chartfield</h3>
 
       <div class="col-sm-3">
         <label translate>Project Id</label>
+        <i
+          class="fa fa-question-circle"
+          button
+          uib-popover="{{'If there is no Project Id (or `NONE`), leave blank' | translate}}"
```

Comment:
I wanted to do ```'If there is no Project Id (or "NONE"), leave blank'```, but the extra set of double quotes around NONE causes an error in the template. Does the same thing for extra single quotes 🙁 I figured having something around it would be better than nothing. There's also the option of defining as a variable ```$scope.projectIdHoverText = 'If there is no Project Id (or "NONE"), leave blank' ``` and then doing ``` {{ projectIdHoverText | translate }}``` but wasn't sure how I felt about that. Let me know what you think.

Reply:
Can you escape the quotes? `\"`

https://stackoverflow.com/questions/22967475/angularjs-html-in-js-escaping-single-quote

I'm not sure what works without trying it.

Reply:
That seems to throw a parse error 🙁

Reply:
Idk then. Sorta a trivial thing. Your current solution is probably fine.